bpf/node_config.h

```diff
@@ -13,14 +13,22 @@
  */
 #include "lib/utils.h"
 
+#ifndef NODE_MAC
```

Comment:
these strictly necessary or are you being defensive?

Reply:
It is actually necessary. We "redefine" this macro in the new [config_replacement.h](https://github.com/dylandreimerink/cilium/blob/c8947a3a061a6b978daace6fe34afcbcc7d4b947/bpf/tests/config_replacement.h#L76). This is necessary since the test framework can't do global data inlineing yet.